utils/ifupdown.sh.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I would prefer to write a POSIX shell script. It's not common to have bash on an embedded system.